src/common/transformations/src/transformations/convert_precision.cpp

```diff
@@ -293,6 +292,11 @@ bool ov::pass::ConvertPrecision::run_on_model(const std::shared_ptr<ngraph::Func
         {opset10::Unique::get_type_info_static(), fuse_type_to_unique_v10},
         {opset8::RandomUniform::get_type_info_static(), fuse_type_to_random_uniform_v8}};
 
+    auto t2f_it = m_additional_type_to_fuse_map.find(opset4::Convert::get_type_info_static());
```

Comment:
I guess we can just override all the values of `type_to_fuse` map, e.g.
```
for (const auto& it : m_additional_type_to_fuse_map) {
      type_to_fuse[it.first] = it.second;
}
```
what do you think?

Reply:
Good point! Fixed. Thanks Ivan!